app/watchers/providers/docker/Docker.js

```diff
@@ -746,6 +746,7 @@ class Docker extends Component {
                 variant,
                 created,
             },
+            composeProject: container.Labels['com.docker.compose.project'],
```

Comment:
One problem I currently see here is that this will only be added to new containers that are not yet in the store